### PR TITLE
Remove pwsh -login and add Pwsh test

### PR DESCRIPTION
**Pull request by JustinGrote**
This resolves the issue with pwsh failing due to -login and the command not splitting correctly.

It also adds a test pwsh run using a image that has Powershell integrated, hopefully this doesn't introduce too much into the testing time, but the image can be cached in the CI if necessary to speed things up.

EDIT: Literally took less than half a second to download and extract the container, I guess that's a benefit of using the github container registry :)

### CODE REVIEW

**Review by catthehacker — APPROVED** [review]

File: pkg/runner/runner_test.go [hunk 1]
```diff
@@ -83,10 +83,14 @@ func TestRunEvent(t *testing.T) {
...
 		"ubuntu-latest": "node:12.20.1-buster-slim",
 	}
+	pwshplatforms := map[string]string{
+		"ubuntu-latest": "ghcr.io/justingrote/act-pwsh:latest",
+	}
 	tables := []TestJobFileInfo{
 		{"testdata", "basic", "push", "", platforms, ""},
 		{"testdata", "fail", "push", "exit with `FAILURE`: 1", platforms, ""},
 		{"testdata", "runs-on", "push", "", platforms, ""},
+		{"testdata", "powershell", "push", "", pwshplatforms, ""},
```
Comment (catthehacker) [inline pkg/runner/runner_test.go:93]: perhaps it would be better as inline map instead of single variable

Comment (JustinGrote) [reply pkg/runner/runner_test.go:93]: @catthehacker I did it expecting more tests to utilize it, but I can make it an inline map if you prefer.

Comment (JustinGrote) [reply pkg/runner/runner_test.go:93]: @catthehacker I did it expecting more tests to utilize it, but I can make it an inline map if you prefer.

Comment (JustinGrote) [reply pkg/runner/runner_test.go]: Fixed in https://github.com/nektos/act/pull/660/commits/a91679c83c55128751c14c6c5bbb6bdaf02c413c

**Comment by JustinGrote** [comment]
@cplee @catthehacker looks like the composite test is flaky on MacOS, maybe a race condition. Empty commit and it worked this time :)

**Comment by cplee** [comment]
@catthehacker - I'd like to test mergify workflow with this one...can you add an `approve` review to this PR?

**Review by catthehacker — APPROVED** [review]

**Review by cplee — APPROVED** [review]

**Comment by cplee** [comment]
@Mergifyio refresh

**Comment by cplee** [comment]
Took some persistence, but Mergify is all dialed in

**Pull request merged**